test/org/opensolaris/opengrok/web/DirectoryListingTest.java

```diff
@@ -101,16 +141,15 @@ private void create() throws Exception {
             assertTrue("Failed to set modification time",
                     file.setLastModified(val));
 
-            if (subdirs == null) {
-                if (size > 0) {
-                    FileOutputStream out = new FileOutputStream(file);
+            if (subdirs == null && size > 0) {
+                try (FileOutputStream out = new FileOutputStream(file)) {
                     byte[] buffer = new byte[size];
                     out.write(buffer);
-                    out.close();
```

Comment:
cause netbeans converted the block to the try-with-resource - it will be closed automatically